cli/lfc/src/test/kotlin/org/lflang/cli/LfcIssueReportingTest.kt

```diff
@@ -155,6 +151,6 @@ class LfcIssueReportingTest {
         trim().replace(Regex("\\s+\\R"), "\n")
             // Replace file path with placeholder. File path is not
             // rendered the same on linux and windows (forward/backward slash)
-            .replace(lfFile.toString(), "%%%PATH.lf%%%")
+            .replace("%%%PATH.lf%%%", lfFile.toString())
```

Comment:
What happened here? It seems like the original intent was to replace the value of `lfFile.toString()` with `"%%%PATH.lf%%%"` and not the other way around.

Reply:
I am not entirely sure either. The test used to fail and worked fine after switching this.